### ISSUE TITLE
Is pending.schema.org/pending.webschemas.org still used for documentation?

### ISSUE THREAD

**Issue by epaulson** [schemaorg/schemaorg#3313]
I am a bit confused by the ['How We Work'](https://schema.org/docs/howwework.html) page and what I deployed to the web, so I wonder if either I'm missing something or if the state of things are nowadays perhaps a bit different than how things used to work, but I don't get what is different on pending.schema.org these days and was hoping someone could clarify. 

As a hypothetical, let's say we wanted to extend Trip to yet another class of trip, this time for Space trips. I'd make an issue (let's 9999) and we'd talk about it, and decide it's a good idea, and someone would make a pull request to add a file as data/ext/pending/issue-9999.ttl with some new turtle and refine that as a community for a while, iterating on the PR.

At some point folks come to consensus and that PR gets merged into the main repo, still with the ttl file as an standalone small fragment living in data/ext/pending/issue-9999.ttl. (This is where another trip extension seems to be at, boattrip from issue 1755)

However, just because it's been merged into the tree hosted at github.com/schemaorg/schemaorg doesn't mean that it's entirely ready to go, and it might stay as a 'pending' item for a couple of releases, e.g. maybe for a year or two - it's part of the 'pending' set of classes and properties in schema.org - we'd have a triple inside of issue-9999.ttl it says thats SpaceTrip `:isPartOf <https://pending.schema.org>`

but once issue-9999.ttl is merged into the tree at github.com.schemaorg/schemaorg, new runs of buildsite.py will pick it up, because that script looks for all .ttl files in data/ and in data/ext/*/*.ttl

This is where I get a little confused - BoatTrip from issue-1755 shows up in the schema.org documentation as https://schema.org/BoatTrip - it's clearly marked as being 'pending' and the ttl that defines it is in data/ext/pending/ and not in data/schema.ttl but the 'How we work' makes it sound like it should only be on https://pending.schema.org/BoatTrip for documentation

I also see BoatTrip in https://schema.org/version/latest/schemaorg-current-https.ttl in the main namespace, though again with an isPartOf pending triple to at least let folks know that it might not become "official"

I haven't exhaustively dug through it, but it does look like pending.schema.org and schema.org have the same content? Maybe in the past that was different?

Some day data/ext/pending/issue-1755.ttl will get deleted and those triples will get moved into data/schema.ttl at which point 'boattrip' will cease to be pending - but I think the only changes that will be noticeable will be a slight change to the documentation that shows up on schema.org/BoatTrip and the 'isPartOf <https://pending.schema.org>' triple will get dropped. 
 
(I also get that webschemas.org is more of a 'nightly build' of schema.org releases - v15 came out in Oct of 2022, it looks like maybe v16 is forthcoming sometime in early summer 2023 - if we had gotten 'SpaceTrip' merged into the schemaorg/schemaorg tree in say December of 2022, you could find it online in https://webschemas.org/SpaceTrip without having to wait 6 months for the next regular release of schema.org - but you shouldn't rely on it because it could get yanked out the next release gets blessed and moved onto schema.org)

**Comment by epaulson** [comment]
Thank you, this was very helpful in better understanding the workflows, and I think your changes have made things easier!

One question - do you think that things will continue on with "new" properties and classes living in small issue-####.ttl files until they eventually get pulled into the main schema.ttl? It seems like it is a nice way to keep merge conflicts minimized while things iterate.

**Comment by epaulson** [comment]
Thanks. I work on a different ontology and I am hoping to steal best practices from schema.org!
